airflow/task/standard_task_runner.py

```diff
@@ -159,15 +158,10 @@ def _start_by_fork(self):
             # [1:] - remove "airflow" from the start of the command
             args = parser.parse_args(self._command[1:])
 
-            # We prefer the job_id passed on the command-line because at this time, the
-            # task instance may not have been updated.
-            job_id = getattr(args, "job_id", self._task_instance.job_id)
             self.log.info("Running: %s", self._command)
-            self.log.info("Job %s: Subtask %s", job_id, self._task_instance.task_id)
+            self.log.info("Subtask %s", self._task_instance.task_id)
```

Comment:
again not clear what is meant by `subtask` it doesn't really tell us anything.  and key-value would be better.

i know it's not your log message just thinking, while we're touching it.